phoenix-core/src/main/java/org/apache/phoenix/coprocessor/MetaDataEndpointImpl.java

```diff
@@ -2925,6 +2954,23 @@ private void clearRemoteTableFromCache(long clientTimeStamp, byte[] schemaName,
         }
     }
 
+    // Checks whether a non-zero PHOENIX_TTL value is being set.
+    private boolean settingNewPhoenixTTLAttribute(List<Mutation> tableMetadata, byte[] phoenixTtlBytes) {
```

Comment:
Can you rename this to `settingNewPhoenixTTLAttributeForView()` to indicate it only applies to views and then also add the check for the same inside the method?